appkit/src/test/scala/org/ergoplatform/appkit/ErgoValueTest.java

```diff
@@ -34,7 +42,11 @@ public void testTypeDeclarations() {
         assertEquals(true, booleanValue);
         assertEquals(1, byteValue);
         assertEquals(1, shortValue);
+        assertFalse(booleanFromCollValue);
         assertEquals(0, byteFromCollValue);
+        assertEquals(0, shortFromCollValue);
+        assertEquals(0, intFromCollValue);
+        assertEquals(0, longFromCollValue);
```

Comment:
I think `0` is not good test vector value, as it often may mean `uninitialized` value.